src/execution.jl

```diff
@@ -539,19 +539,39 @@ end
     @ballocated expression [other parameters...]
 
 Similar to the `@allocated` macro included with Julia,
-this returns the number of bytes allocated when executing
+this returns the *number of bytes allocated* when executing
 a given expression.   It uses the `@benchmark`
 macro, however, and accepts all of the same additional
 parameters as `@benchmark`.  The returned allocations
 correspond to the trial with the *minimum* elapsed time measured
 during the benchmark.
+
+See also `@ballocations`.
 """
 macro ballocated(args...)
     return esc(quote
         $BenchmarkTools.memory($BenchmarkTools.minimum($BenchmarkTools.@benchmark $(args...)))
     end)
 end
 
+"""
+    @ballocations expression [other parameters...]
+
+Similar to the `@allocs` macro included with Julia (v1.9+),
+this returns the *number of allocations* when executing
+a given expression. It uses the `@benchmark` macro, however,
+and accepts all of the same additional parameters as `@benchmark`.
+The returned allocations correspond to the trial with
+the *minimum* elapsed time measured during the benchmark.
+
```

Comment:
Can we set a default value `evals=1 samples=2` for this one? And explain it in the docstring?

Reply:
Then again it would be incoherent with the other `@bstuff` macros so I'm torn